docs/configuration/features/image_features.md

```diff
@@ -1,76 +1,340 @@
+
+- Default: 32
+
+### `kernel_size`
+
+If a `kernel_size` is not already specified in `conv_layers`, specifies the default `kernel_size` of the 2D convolutional kernel that will be used for each layer.
+
+This can be a single integer or a tuple of integers. If single integer, then the kernel has the same dimension in both height and width. If a tuple of integers, then the first integer is the height and the second integer is the width.
+
+- Default: 3
+
+### `stride`
+
+If a `stride` is not already specified in `conv_layers`, specifies the default `stride` of the 2D convolutional kernel that will be used for each layer.
+
+This can be a single integer or a tuple of integers. If single integer, then the kernel has the same dimension in both height and width. If a tuple of integers, then the first integer is the height and the second integer is the width.
+
+- Default: 1
+
+### `padding`
+
+If `padding` is not already specified in `conv_layers`, specifies the default `padding` of the 2D convolutional kernel that will be used for each layer.
+
+- Default: `valid`
+- Choices: `valid`, `same`
+
+### `dilation`
+
+If `dilation` is not already specified in `conv_layers`, specifies the default `dilation` of the 2D convolutional kernel that will be used for each layer.
+
+- Default: `(1, 1)`
+
+### `conv_bias`
+
+If `bias` not already specified in `conv_layers`, specifies if the 2D convolutional kernel should have a bias term.
+
+- Default: `true`
+
+### `conv_weights_initializer`
+
+ (default `'glorot_uniform'`): initializer for the weights matrix. Options are: `constant`, `identity`, `zeros`, `ones`, `orthogonal`, `normal`, `uniform`, `truncated_normal`, `variance_scaling`, `glorot_normal`, `glorot_uniform`, `xavier_normal`, `xavier_uniform`, `he_normal`, `he_uniform`, `lecun_normal`, `lecun_uniform`. Alternatively it is possible to specify a dictionary with a key `type` that identifies the type of initializer and other keys for its parameters, e.g. `{type: normal, mean: 0, stddev: 0}`. To know the parameters of each initializer, please refer to [TensorFlow's documentation](https://www.tensorflow.org/api_docs/python/tf/keras/initializers).
+
+### `conv_bias_initializer`
+
+ (default `'zeros'`):  initializer for the bias vector. Options are: `constant`, `identity`, `zeros`, `ones`, `orthogonal`, `normal`, `uniform`, `truncated_normal`, `variance_scaling`, `glorot_normal`, `glorot_uniform`, `xavier_normal`, `xavier_uniform`, `he_normal`, `he_uniform`, `lecun_normal`, `lecun_uniform`. Alternatively it is possible to specify a dictionary with a key `type` that identifies the type of initializer and other keys for its parameters, e.g. `{type: normal, mean: 0, stddev: 0}`. To know the parameters of each initializer, please refer to [TensorFlow's documentation](https://www.tensorflow.org/api_docs/python/tf/keras/initializers).
+
+### `weights_regularizer`
```

Comment:
The *_regularizer parameters (`weights_regularizer`, `bias_regularizer`...) for layers are gone as of 0.5 - remove

Reply:
@dantreiman thanks for pointing that out, I've removed the references to conv regularizers.

I did a quick skim through the code and it looks like that there are still fully connected weight regularizers, so I've kept those in for now.